app/models/office.rb

```diff
@@ -43,7 +43,7 @@ class Office < ActiveRecord::Base
   after_commit :notify_indexer
 
   def notify_indexer
-    UpdateAlgoliaIndexJob.perform_async(model_name.name, id, firm_id)
+    UpdateAlgoliaIndexJob.perform_later(model_name.name, id, firm_id)
```

Comment:
Just saying Margo's PR #454 also changed this line. Just saying :)

Reply:
yeah we were syncing on how to fix multiple test issues :sweat_smile: . Will be rebased against this (if this gets in first as I guess). Thanks for the check!